src/boilerplate/common/services/genericpublic-api_services.mjs

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable prettier/prettier, camelcase, prefer-const, no-unused-vars */
+import config from "config";
+import assert from "assert";
+import FUNCTION_NAME from './FUNCTION_NAME.mjs';
+import fs from "fs";
+import logger from "./common/logger.mjs";
+import web3 from "./common/web3.mjs";
+const sleep = (ms) => new Promise((r) => setTimeout(r, ms));
+let leafIndex;
+let encryption = {};
+// eslint-disable-next-line func-names
+export async function service_FUNCTION_NAME (req, res, next){
+  try {
+    await web3.connect();
+    await new Promise((resolve) => setTimeout(() => resolve(), 3000));
```

Comment:
This is might not be needed as `await` instruction was used in the previous line.